### PR TITLE
Added documentations for prometheus

### PR DESCRIPTION
**Pull request by goruha**
## What
* Provide documentation about prometheus
* Answer to question https://github.com/cloudposse/docs/issues/31

## Why
* Making documentation

### CODE REVIEW

File: content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
 description: ""
-draft: true
 ---
+ [Prometheus](https://prometheus.io) is the heart of monitoring backing service. The other parts are [Alert Manager](https://prometheus.io/docs/alerting/alertmanager/), [Grafana](https://grafana.com/) and wide list of [exporters](https://prometheus.io/docs/instrumenting/exporters/) that are source of metric.
+
+{{< img src="/assets/324asd-Prometheus_architecture.png" title="Architecture" >}}
+
+# Prometheus
+
+Prometheus - is monitoring system and time-series database.
```
Comment (osterman) [inline content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md]: Doesn't it use InfluxDB as the time series database?

File: content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md [hunk 2]
```diff
@@ -1,5 +1,69 @@
...
+
+Prometheus pull metrics data from different exporters and put that data to the storage.
+To get list of available exporters Prometheus use discovery service where available exporters should be registred.
+
+It have list of registred alerts. [Alert](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) is a rule with describes alert conditions based on Prometheus expression language expressions.
+When alert conditions became true Prometheus sends alert notification to Alert Manager.
+
+Also Prometheus provide endpoint to query requests for metrics data. Grafana use this endpoint to get data for charts.
+
+# Exporter
```
Comment (osterman) [inline content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md]: Isn't the official name `Prometheus Exporter`? 

We can write it like 
```
# Prometheus Exporter (aka Exporter)
```

File: content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md [hunk 3]
```diff
@@ -1,5 +1,68 @@
...
+
+# Prometheus
+
+Prometheus is monitoring system and time-series database.
+
+It is responsible for several things:
+* Scrapping metrics data from exporters
+* Storing metrics data
+* Firing alerts
+* Response to queries
```
Comment (osterman) [inline content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md]: Can you explain what this is or reword it? "response to queries" - I don't know what it refers to

Comment (goruha) [reply content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md]: You can select metrics data by sending http request with query

File: content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md [hunk 4]
```diff
@@ -1,5 +1,69 @@
 ---
 title: "Prometheus, Alerts & Grafana"
-description: ""
-draft: true
+description: "Prometheus is monitoring system and time-series database that works together with Grafana and Alert Manage to provide an end-to-end monitoring platform for Kubernetes."
```
Comment (aknysh) [inline content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md]: Alert Manager

File: content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md [hunk 5]
```diff
@@ -1,5 +1,69 @@
...
 ---
+ [Prometheus](https://prometheus.io) is at the heart of the monitoring infrastructure. It works in concert with [Alert Manager](https://prometheus.io/docs/alerting/alertmanager/), [Grafana](https://grafana.com/) and wide list of [Exporters](https://prometheus.io/docs/instrumenting/exporters/) that provide the metrics for instrumentation.
+
+{{< img src="/assets/324asd-Prometheus_architecture.png" title="Prometheus Monitoring Architecture" >}}
+
+# Prometheus
+
+Prometheus is monitoring system and time-series database.
```
Comment (aknysh) [inline content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md]: a monitoring system

File: content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md [hunk 6]
```diff
@@ -1,5 +1,69 @@
...
+The Prometheus API service is used by other services to query for metrics data. For example, Grafana uses this endpoint to fetch the data needed by charts.
+
+# Prometheus Exporter (aka Exporter)
+
+The Prometheus Exporter is an http service that exposes metrics in a format suitable for Prometheus consumption.
+
+Sometimes this will be a simple service like the
+[Node exporter](https://github.com/prometheus/node_exporter) which provides some basic server metrics (e.g. CPU/Memory/etc...)
+
+Othertimes, software exposes metrics in a prometheus-compatable format out-of-the-box. For a list of software the integrates natively with Prometheus, check out the [list of software exposing prometheus metrics](https://prometheus.io/docs/instrumenting/exporters/#software-exposing-prometheus-metrics)
```
Comment (aknysh) [inline content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md]: that integrates

**Pull request merged**